### PR TITLE
meta dump tool

### PR DESCRIPTION
**Pull request by critical27**
Add a tool to dump important meta infos (space/part/schema/hosts...).
We only need package these tools `db_dump/db_upgrader/meta_dump`.

### CODE REVIEW

**Review by panda-sheep — COMMENTED** [review]
Good job！

File: src/tools/simple-kv-verify/CMakeLists.txt [hunk 1]
```diff
@@ -45,11 +45,11 @@ nebula_add_executable(
...
-        tool
-)
+#install(
+#    TARGETS
+#        simple_kv_verify
+#    DESTINATION
+#        bin
+#    COMPONENT
+#        tool
+#)
```
Comment (panda-sheep) [inline src/tools/simple-kv-verify/CMakeLists.txt:55]: These tools can be compiled？
Just don’t include them when packing.

Comment (critical27) [reply src/tools/simple-kv-verify/CMakeLists.txt:55]: Yeah, they will be compiled but not packaged.

File: src/tools/meta-dump/MetaDumpTool.cpp [hunk 2]
```diff
@@ -0,0 +1,200 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
```
Comment (laura-ding) [inline src/tools/meta-dump/MetaDumpTool.cpp:1]: 2021

File: src/tools/meta-dump/MetaDumpTool.cpp [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+#include "meta/MetaServiceUtils.h"
+
+DEFINE_string(path, "", "rocksdb instance path");
+
+namespace nebula {
+namespace meta {
+
+class MetaDumper {
+public:
+    static void dumpMeta(const char* path) {
```
Comment (laura-ding) [inline src/tools/meta-dump/MetaDumpTool.cpp:20]: return the Status

File: src/tools/meta-dump/MetaDumpTool.cpp [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+        LOG(INFO) << "open rocksdb on " << path;
+        rocksdb::DB* db = nullptr;
+        rocksdb::Options options;
+        auto status = rocksdb::DB::OpenForReadOnly(options, path, &db);
+        CHECK(status.ok()) << status.ToString();
```
Comment (laura-ding) [inline src/tools/meta-dump/MetaDumpTool.cpp:25]: CHECK will generate core file, it's not necessary

File: src/tools/meta-dump/MetaDumpTool.cpp [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+        rocksdb::ReadOptions roptions;
+        rocksdb::Iterator* iter = db->NewIterator(roptions);
+        if (!iter) {
+            LOG(FATAL) << "null iterator!";
```
Comment (laura-ding) [inline src/tools/meta-dump/MetaDumpTool.cpp:29]: ditto

**Review by panda-sheep — APPROVED** [review]

**Review by bright-starry-sky — APPROVED** [review]
Well done.

**Review by laura-ding — APPROVED** [review]

**Pull request merged**